test/runnable/xtest46.d

```diff
@@ -3019,15 +3020,15 @@ struct Perm {
         foreach(elem; input) {
             enforce136(i < 3);
             perm[i++] = elem;
-            std.stdio.stderr.writeln(i);  // Never gets incremented.  Stays at 0.
+            printf("%d\n", i);  // Never gets incremented.  Stays at 0.
```

Comment:
`fprintf()` to `stderr`?  Does it matter?

Reply:
no